imod/msw/infiltration.py

```diff
@@ -74,3 +99,28 @@ def __init__(
         self.dataset["extra_storage_coefficient"] = extra_storage_coefficient
 
         self._pkgcheck()
+
+    @classmethod
+    def from_imod5_data(cls, imod5_data: dict[str, GridDataDict]) -> "Infiltration":
+        cap_data = imod5_data["cap"]
+        data = {}
+        # Use runon resistance as downward resistance, and runoff for downward
```

Comment:
I think run-on needs to have a dash in it. At least that's what i find online. The weird thing is that it is not needed for runoff

Reply:
Weird! I think I prefer keeping the naming consistent (i.e. "runon" and "runoff"), as I like consistency and python variable names cannot contain a dash